x/concentrated-liquidity/fees.go

```diff
@@ -0,0 +1,39 @@
+package concentrated_liquidity
+
+import (
+	"strconv"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/osmoutils/accum"
+)
+
+const (
+	feeAccumPrefix        = "fee"
+	feeAccumNameSeparator = "/"
+	uintBase              = 10
+)
+
+func (k Keeper) createFeeAccumulator(ctx sdk.Context, poolId uint64) error {
```

Comment:
can you please add godocs to all methods

Reply:
Added